core/domain/event_services_test.py

```diff
@@ -436,6 +437,39 @@ def test_stats_events_successfully_updated(self) -> None:
         self.assertEqual(model.exp_id, exp_id)
         self.assertEqual(model.exp_version, exploration.version)
 
+    def test_cannot_update_stats_events_for_older_exp_version(self) -> None:
+        all_models = (
+            stats_models.ExplorationStatsModel.get_all())
+        self.assertEqual(all_models.count(), 0)
+
+        exp_id = 'eid1'
+        self.save_new_valid_exploration(exp_id, self.OWNER_EMAIL)
+        exp_services.update_exploration(
+            self.OWNER_EMAIL, exp_id, [exp_domain.ExplorationChange({
+            'new_value': {
+                'content_id': 'content',
+                'html': 'content 1'
+            },
+            'state_name': 'Introduction',
+            'old_value': {
+                'content_id': 'content',
+                'html': ''
+            },
+            'cmd': 'edit_state_property',
+            'property_name': 'content'
+            })], 'Update 1')
```

Comment:
This indentation is confusing because the dictionary elements are not at the same "level" as function args like `self.OWNER_EMAIL`

Reply:
Done